crates/ruff_python_formatter/resources/test/fixtures/black/py_312/type_aliases.py

```diff
@@ -0,0 +1,5 @@
+type A=int
+type Gen[T]=list[T]
+
+type = aliased
+print(type(42))
```

Comment:
Is this covering use as an identifier? Many more cases at https://github.com/RustPython/Parser/blob/704eb40108239a8faf9bd1d4217e8dad0ac7edb3/parser/src/parser.rs#L945-L974 but honestly I'm not sure these belong in the formatter tests unless we can't assume the parser is handling this correctly?